client/keys/list.go

```diff
@@ -6,15 +6,15 @@ import (
 	"github.com/spf13/cobra"
 )
 
-// CMD
-
-// listKeysCmd represents the list command
-var listKeysCmd = &cobra.Command{
-	Use:   "list",
-	Short: "List all keys",
-	Long: `Return a list of all public keys stored by this key manager
+func listKeysCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List all keys",
+		Long: `Return a list of all public keys stored by this key manager
```

Comment:
One line plz

Reply:
I am not sure what you mean here

Reply:
oh! I guess is that you avoid the variable and return straight away. :+1: